stubs/Flask-Migrate/METADATA.toml

```diff
@@ -1,6 +1,6 @@
 version = "4.0.*"
-# Requires a version of flask with a `py.typed` file
-requires = ["Flask>=2.0.0", "types-Flask-SQLAlchemy"]
+# Requires versions of flask/Flask-SQLAlchemy with `py.typed` files
+requires = ["Flask>=2.0.0", "Flask-SQLAlchemy>=3.0.1"]
```

Comment:
I figured it out. Because you're specifying Flask-SQLAlchemy as a dependency of types-Flask-Migrate, that then pulls in SQLAlchemy as a transitive dependency of types-Flask-Migrate (because SQLAlchemy is a dependency of Flask-SQLAlchemy). That means pip now install SQLAlchemy in CI before pyright tests any of the stubs and, because it's unpinned, pip installs the latest version of SQLAlchemy available. Because `SQLAlchemy>=2` has inline types and a `py.typed` file, _that_ means that pyright is now pointing out the inconsistencies between our stubs for `SQLAlchemy<2` and the inline types in site-packages for `SQLAlchemy>=2`

Reply:
We could just add the missing generics.

Reply:
> We could just add the missing generics. So that the stubs are on par enough with the 2.0 version for this PR to pass.

It feels like a somewhat fragile solution -- if `SQLAlchemy` makes more stuff generic on the 2.0 branch that isn't generic in our stubs for `SQLAlchemy` 1.0, and they then make a release, our CI will break unexpectedly

Reply:
Then I think just waiting to remove them together, given their co-dependencies, just makes the most sense.